scripts/windows/VERIFICATION.txt

```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+ 
+To verify the logdna-agent.exe:
```

Comment:
Can you remove the `the` here? Just "To verify logdna-agent.exe:"

Reply:
Done!